### ISSUE TITLE
#2037 follow up, cg vs. cgo

### ISSUE THREAD

**Issue by zonkzonk** [radare/radare2#2496]
morrn,

 a #2037 follow up

https://asciinema.org/a/19561

can anyone else reproduce?

Greetings
--zlul

```
zlul@debian:~$  uname -a
Linux debian 3.2.0-4-686-pae #1 SMP Debian 3.2.68-1+deb7u1 i686 GNU/Linux
zlul@debian:~$ cat /etc/debian_version 
7.8
```

**Comment by crowell** [comment]
yep, segfaults on amd64 for me

**Comment by crowell** [comment]
fixed in 08953547746314c296078075252a0ce0904d1153

**Comment by zonkzonk** [comment]
@crowell many thanks for fixing.
